plugins/google_drive/help.md

```diff
@@ -14,87 +14,204 @@ This plugin utilizes the [Google Drive API](https://developers.google.com/drive/
 * A JWT With Google Drive Permissions
 * The Google Drive API must be enabled
 
+# Supported Product Versions
+
+_There are no supported product versions listed._
+
 # Documentation
 
 ## Setup
 
 The connection configuration accepts the following parameters:
 
-|Name|Type|Default|Required|Description|Enum|
-|----|----|-------|--------|-----------|----|
-|private_key|password|None|True|Private Key from service credentials|None|
-|admin_user|string|None|True|Admin user to impersonate, e.g. user@example.com|None|
-|private_key_id|password|None|True|Private Key ID from service credentials|None|
-|token_uri|string|https://accounts.google.com/o/oauth2/token|True|OAUTH2 Token URI|None|
-|auth_provider_x509_cert_url|string|https://www.googleapis.com/oauth2/v1/certs|True|OAUTH2 Auth Provider x509 Cert URL|None|
-|auth_uri|string|https://accounts.google.com/o/oauth2/auth|True|None|None|
-|client_email|string|None|True|Client email from service credentials|None|
-|client_id|string|None|True|Client ID|None|
-|project_id|string|None|True|Project ID from service credentials|None|
-|client_x509_cert_url|string|None|True|x509 cert URL from service credentials|None|
+|Name|Type|Default|Required|Description|Enum|Example|
+|----|----|-------|--------|-----------|----|-------|
+|admin_user|string|None|True|Admin user to impersonate, e.g. admin@domain.com|None|admin@domain.com|
```

Comment:
```suggestion
|admin_user|string|None|True|Admin user to impersonate, e.g. user@example.com|None|user@example.com|
```
The ConfidentialValidator is complaining about these example emails. If we want to satisfy the validator we should replace them with these emails, however we should discuss if we may want to ignore the validator and keep these examples to show that an administrator is required.